SushiRunner.Services/MealService.cs

```diff
@@ -45,6 +53,20 @@ public void Delete(long id)
             _repository.Save();
         }
 
+        public IEnumerable<MealDTO> GetByGroup(string groupName)
+        {
+            var meals = _repository.Search(m => m.MealGroup.Name.Equals(groupName));
+            
+            return meals.Select(meal => _mapper.Map<Meal, MealDTO>(meal)).ToList();
```

Comment:
Maybe just:

```
return _repository.Search(...)
    .Select(meal => _mapper.Map<Meal, MealDTO>(meal))
    .ToList();
```

Reply:
Oles! Do you remember any Svyatoslav Volodymyrovych lessons? One action - one row.